### ISSUE TITLE
Add volumes example and instruction for photon volume plugin

### ISSUE THREAD

**Issue by luomiao** [vmware/kubernetes#72]
Examples should be added at:
https://github.com/kubernetes/kubernetes/tree/master/examples/volumes

Docs:
https://kubernetes.io/docs/user-guide/volumes/
https://kubernetes.io/docs/user-guide/persistent-volumes/

**Comment by AlainRoy** [comment]
Should we also document this in the Photon Controller documentation? Or at least point to this documentation?

**Comment by pdhamdhere** [comment]
I agree. Photon Controller K8S User Guide (if exists) would be better place than K8S.
